### DISCUSSION TITLE
ilogtail 2.0编译后体积太大

### DISCUSSION THREAD

**Discussion by daggerjames** [alibaba/ilogtail#1570] [Help] [answered]
源码编译的情况下，编译出的ilogtail体积有1.5G, 但是pre-built下的ilogtail只有20M，想知道是啥原因。
编译的方式完全按照教程走的，挂镜像进去按照
https://ilogtail.gitbook.io/ilogtail-docs/developer-guide/development-environment#wef4t
执行。和make core没区别

**Comment by messixukejia** [comment, marked as answer]
因为编出来的binary有符号表，需要使用strip命令剥离。

![image](https://github.com/alibaba/ilogtail/assets/13250519/4568b1d4-0173-4dc9-9e81-fa75c0a2df8d)

欢迎帮忙优化下文档吧
